shared/components/theme/more-button.jsx

```diff
@@ -69,7 +73,7 @@ var ThemeMoreButton = React.createClass( {
 						if ( option.url ) {
 							return (
 								<a className="theme__more-button-menu-item popover__menu-item"
-									onMouseOver={ event => {
+									onMouseOver={ function( event ) {
```

Comment:
TBH, I wasn't so sure myself either, so I was hoping for a comment like yours :-)
I'll turn it into a method.